Website/plugins/secondaries/gen-secondaries.raku

```diff
@@ -1,27 +1,51 @@
 #!/usr/bin/env raku
 use v6.d;
+use RakuConfig;
 use ProcessedPod;
 use Collection::Progress;
+use nqp;
 
-sub ($pp, %processed, %options) {
-    # these chars cannot appear in a unix filesystem path
+sub (ProcessedPod $pp, %processed, %options) {
     my regex defnmark {
         '<!-- defnmark' \s+
         $<target> = (.+?) \s+
+        $<level> = (\d+) \s+
         '-->'
-        $<body> = (.+?)
-        <?before '<h' | '</section' | '</body' | $ >
+    }
+    my $level;
+    my regex chunk {
+        (.+?)
+        <?before
+            "<h$level"
+            | '</section'
+            | '</body'
+            | $
+        >
     }
     sub good-name($name is copy --> Str) is export {
+    # these chars cannot appear in a unix filesystem path
+        # Documentable code
         # / => $SOLIDUS
         # % => $PERCENT_SIGN
         # ^ => $CIRCUMFLEX_ACCENT
         # # => $NUMBER_SIGN
+        #    my @badchars  = ["/", "^", "%"];
+        #    my @goodchars = @badchars
+        #                    .map({ '$' ~ .uniname      })
+        #                    .map({ .subst(' ', '_', :g)});
+        #
+        #    $name = $name.subst(@badchars[0], @goodchars[0], :g);
+        #    $name = $name.subst(@badchars[1], @goodchars[1], :g);
+        #    # if it contains escaped sequences (like %20) we do not
+        #    # escape %
+        #    if ( ! ($name ~~ /\%<xdigit>**2/) ) {
+        #        $name = $name.subst(@badchars[2], @goodchars[2], :g);
+        #    }
```

Comment:
Same comment about commented out code.

Reply:
Here the commented code come from Documentable. 
The `Website/secondaries` plugin has to match the output from Documentable in order to conserve all the `Raku/doc/` source references. 
I included it to show why there are some odd choices in the coding for this plugin.

Reply:
deleting redundant comments